src/cascadia/TerminalConnection/ConptyConnection.h

```diff
@@ -93,6 +97,14 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         std::array<char, 4096> _buffer{};
         bool _passthroughMode{};
 
+        struct StartupInfoFromDefTerm
```

Comment:
whoa, type declaration and a member declaration at the same time!

Reply:
Yea, I thought this was an appropriate time to use this - is it not/never?

Reply:
Fine with me, just rare to see in the wild. :)